resources/keycloak/README.md

```diff
@@ -10,3 +10,67 @@ The `realm.json` file included in this project initializes Keycloak with a sampl
 ## Keycloak Theme
 
 A sample keycloak theme is provided in the `sample_theme.jar` file. This is a sample theme generated using [Keycloakify](https://github.com/CDOT-CV/keycloakify-starter), to use a custom theme put a generated .jar file in this directory and then update the `KEYCLOAK_LOGIN_THEME_NAME` with the name of the new .jar file.
+
+## Migration Steps
+
+This section describes the steps required to add this custom user provider to an existing cvmanager deployment. If followed correctly, there will be no action required by users (other than possible having local users re-set their credentials, more on that later), and no user data will be lost.

```

Comment:
"other than possible having local users" is this meant to be "possibly"

Reply:
fixed